src/elm/Page/Shop/Editor.elm

```diff
@@ -340,14 +340,15 @@ viewForm shared balances imageStatus isEdit isDisabled deleteModal form =
                         , disabled isDisabled
                         ]
                         []
+                    , viewFieldErrors (listErrors shared.translations form.title)
                     ]
                 , formField
                     [ div
-                        [ class "input-label" ]
+                        [ class "input-label mt-3" ]
                         [ text (t "shop.description_label") ]
-                    , viewFieldErrors (listErrors shared.translations form.description)
                     , textarea
-                        [ class "input w-full"
+                        [ class "input w-full mb-0"
+                        , classList [ ( "border-red ", hasErrors form.description ) ]
```

Comment:
There's [`.field-with-error`](https://github.com/cambiatus/frontend/blob/master/src/styles/main.css#L116) class in the `main.css`. Please, use it instead of just a red border. This class also makes the text red as in the [Figma form elements mock-up](https://www.figma.com/file/oZJUIyHRmLyLIKDxYebvLg/Form-elements?node-id=0%3A1).